### PR TITLE
removed service client from interface

### PR DESCRIPTION
**Pull request by marc-hessenauer**
changed as discussed in the docs pr

https://github.com/fraym/docs/pull/7

### CODE REVIEW

File: .DS_Store
Comment (jesko-plitt) [inline .DS_Store:1]: please add this file to the .gitignore

File: src/client/client.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export interface Client {
         handler: HandlerFunc
     ) => Promise<void>;
     publish: (topic: string, events: PublishEvent[]) => Promise<void>;
-    getStreamItarator: (
+    getStreamIterator: (
```
Comment (jesko-plitt) [inline src/client/client.ts:40]: we have to update the changelog.md and add a breaking change (see https://release-notes.becklyn.tech/ for a reference on entries)

**Review by jesko-plitt — CHANGES_REQUESTED** [review]

File: src/client/client.ts (hunk 1 above)
Comment (marc-hessenauer) [reply src/client/client.ts:40]: done

File: .DS_Store
Comment (marc-hessenauer) [reply .DS_Store:1]: added the file name on root level with a newline as follows:
.DS_Store

Comment (jesko-plitt) [inline .DS_Store:1]: please remove the file from the commit, too

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # vNext
 
+-   (bc) renamed getStreamItarator to getStreamIterator
```
Comment (jesko-plitt) [inline CHANGELOG.md:3]:
```suggestion
-   (bc) Rename `getStreamItarator` to `getStreamIterator`
```

**Review by jesko-plitt — CHANGES_REQUESTED** [review]

**Review by jesko-plitt — APPROVED** [review]

**Pull request merged**